CodeReview/src/commands/CFocus.cpp

```diff
@@ -61,41 +63,44 @@ bool CFocus::canInterpret(Visualization::Item*, Visualization::Item*,
 Interaction::CommandResult* CFocus::execute(Visualization::Item*, Visualization::Item*,
 				const QStringList&, const std::unique_ptr<Visualization::Cursor>&)
 {
+	focusStep({}, {}, {});
+	return new Interaction::CommandResult{};
+}
 
+bool CFocus::focusStep(Visualization::Item *, QKeySequence, Interaction::ActionRegistry::InputState)
+{
```

Comment:
I also use `Visualization::VisualizationManager::instance().mainView()->centerOn(focusItem);`. Is there also a way to do this using target?